default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib
+, runCommand
+, poetry2nix
+, python3
+, rustPlatform
+, yara
+, _7zz
+, lz4
+, lziprecover
+, lzo
+, lzop
+, simg2img
+, squashfsTools
+, unar
+}:
+
+let
+  # These dependencies are only added to PATH
+  runtimeDeps = [
+    _7z
+    lz4
+    lziprecover
+    lzop
+    simg2img
+    squashfsTools
+    unar
+  ];
+
+  _7z = runCommand "7z" { } ''
+    mkdir -p $out/bin
+    ln -snf ${_7zz}/bin/7zz $out/bin/7z
+  '';
+
+  self = poetry2nix.mkPoetryApplication {
+    projectDir = ./.;
+
+    # Python dependencies that need special care, like non-python
+    # build dependencies
+    overrides = poetry2nix.overrides.withDefaults (self: super: {
+      python-lzo = super.python-lzo.overridePythonAttrs (_: {
+        buildInputs = [
+          lzo
+        ];
+      });
+
+      jefferson = super.jefferson.overridePythonAttrs (_: {
+        propagatedBuildInputs = [
+          # Use the _same_ version as unblob
+          self.cstruct
+        ];
+      });
+
+      yara-python = super.yara-python.overridePythonAttrs (_: {
+        # Use _our_ patched version of yara
```

Comment:
Don't know where to put this, but general question / concerns about patching: 

* Is this a common practice to patch dependencies "locally" like this? 
* Do you consider this as a temporary work-around, so we should get rid of this and update the dependencies as soon as we can?

Reply:
Yes, given a new fixed release is available from these tools, we can get rid of these specific patches. On the other hand custom gnustep will be with us for quite some time I guess.